frontend/public/style/_overrides.scss

```diff
@@ -119,6 +119,13 @@ h6 {
   }
 }
 
+.pf-c-button.pf-m-inline.pf-m-link.co-dashboard-card__button-link {
```

Comment:
You don't need `isInline` in the jsx and `padding: 0`.  Are all these declarations needed with the PF4 button styles?

I prefer keeping this code with the component (in frontend/public/components/dashboard/dashboard-card/card.scss).  I'd remove the extra specificity and add `!important` where needed.

Reply:
updated